rewrite-core/src/main/java/org/openrewrite/config/ClasspathScanningLoader.java

```diff
@@ -116,6 +116,7 @@ private void scanYaml(ClassGraph classGraph, Properties properties, @Nullable Cl
     private void scanClasses(ClassGraph classGraph, ClassLoader classLoader) {
         try (ScanResult result = classGraph
                 .ignoreClassVisibility()
+                .overrideClassLoaders(classLoader)
```

Comment:
I know you were fixing a class loader issue early this week, is this just a missed case that you are including with the PR?

Reply:
Didn't _intend_ to include it in this PR, but it does fix a long standing oversight